contracts/interfaces/clearinghouse/IClearingHouseCustomErrors.sol

```diff
@@ -54,4 +54,7 @@ interface IClearingHouseCustomErrors is IClearingHouseStructures {
 
     /// @notice error to denote that keeper fee is negative or zero
     error KeeperFeeNotPositive(int256 keeperFee);
+
+    /// @notice error to denote an invalid setting for parameters
+    error InvalidSetting(bytes32 errorMessage);
```

Comment:
From the analysis of bytecode size, it would work if we use `InvalidSetting(uint256)` errors.